pkg/cmd/kafka/topic/list/list.go

```diff
@@ -122,6 +144,14 @@ func runCmd(opts *Options) error {
 		a = a.Filter(opts.search)
 	}
 
+	if opts.size != defaultSize {
+		a = a.Size(int32(opts.size))
+	}
+
+	if opts.page != defaultPage {
```

Comment:
These defaults are returned by the response when we don't set flags manually.
![Screenshot from 2021-07-08 18-22-18](https://user-images.githubusercontent.com/23582438/124924988-a14e4e80-e019-11eb-8063-f88ddfc86968.png)
Should we let go of that check?

Reply:
If the backend changes the default size to 20 for example, the CLI will continue to use 10 as the default value. The user may enter `--size=10` as the page size, but pagination will not be effective because the comparison is happening against the local `defaultSize` value.

I think we should have a single, common `defaultSize` which can be used in control plane and data plane APIs. This gives the CLI more control over pagination defaults and can ensure consistency when different APIs have different defaults.

Make sense?

Reply:
Also - think about upstream. Default value of 10 is an implementation detail of RHOAS service APIs, but if this CLI could be reused within a different environment upstream those defaults could be completely different.